templates/mathjax_config.html

```diff
@@ -19,3 +11,28 @@
         messageStyle: "none",
     });
 </script>
+<script type="application/javascript">
+    // When Mathjax has been loaded (asynchronously)
+    document.querySelector("#mathjax-script").addEventListener("load", function() {
+
+        // We look for <span> inside contents
+        $(".message-content span, .article-content span").each(function () {
+            // We only want those with no CSS class
+            if($(this).attr("class") == undefined) {
```

Comment:
`== undefined` ça a un comportement correct? surtout, le linter accepte un `==`? faudrait pas mieux un `!$(this).classes()`

Reply:
Oui car `$(this).attr("class")` retourne `undefined` quand il n'y a aucune classe CSS. Le linter JS ne fonctionne pas ici car on est dans le dossier `templates`. J'ai mis l'égalité `===` qui est plus stricte que `==` normalement. `$(this).classes()` n'existe pas.